modules/effects/src/effect_decorator.ts

```diff
@@ -4,12 +4,22 @@ import { getSourceForInstance } from './utils';
 
 const METADATA_KEY = '__@ngrx/effects__';
 
-export function Effect<T>({ dispatch = true } = {}): PropertyDecorator {
+export function Effect<T>({
+  dispatch = true,
```

Comment:
Could we make use of the `EffectConfig` model here?

Reply:
Done. Also I'd love to take a second look how and where these defaults are set.

Reply:
Do you want to do this in this PR or do it in a separate PR?